### PR TITLE
dabest recipe

### PR DESCRIPTION
**Pull request by josesho**

### CODE REVIEW

File: recipes/DABEST/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    - python
+    - setuptools
+  run:
+    - python
+    - matplotlib
+    - seaborn
+    - pandas
+    - numpy
+    - scipy
+    - pytest
```
Comment (synapticarbors) [inline recipes/DABEST/meta.yaml]: It doesn't look like `pytest` is an actual run time requirement of this package.

File: recipes/DABEST/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
```
Comment (synapticarbors) [inline recipes/DABEST/meta.yaml]: Are there any version restrictions on any of the `run` requirements? It looks like there are specific versions mentioned in the `setup.py`, but maybe they're not strict limits.

File: recipes/DABEST/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    - python
+    - matplotlib
+    - seaborn
+    - pandas
+    - numpy
+    - scipy
+    - pytest
+
+test:
+  imports:
```
Comment (synapticarbors) [inline recipes/DABEST/meta.yaml]: You should only test the import of the package in the recipe, not its dependencies. Currently you're not doing the latter and only the former. It should look like:

```
tests:
   imports:
      - dabest
```

File: recipes/DABEST/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    - sys
+    - numpy
+    - scipy
+    - pandas
+    - seaborn
+
+about:
+  home: https://acclab.github.io/DABEST-python-docs/index.html
+  license: 'BSD 3-clause'
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (synapticarbors) [inline recipes/DABEST/meta.yaml]: The license should probably be packaged with the source distribution. For now, you could include the following link in the recipe as a comment:

https://github.com/ACCLAB/DABEST-python/issues/12

File: recipes/DABEST/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```
Comment (synapticarbors) [inline recipes/DABEST/meta.yaml]: Also, this package might be a candidate for `noarch: python`

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages

File: recipes/DABEST/meta.yaml (hunk 4 above)
Comment (josesho) [reply recipes/DABEST/meta.yaml]: Done, thanks for the heads up!

File: recipes/DABEST/meta.yaml (hunk 2 above)
Comment (josesho) [reply recipes/DABEST/meta.yaml]: it is for a few of the packages; should I add something?

File: recipes/DABEST/meta.yaml (hunk 5 above)
Comment (josesho) [reply recipes/DABEST/meta.yaml]: Ah, thanks for this!

File: recipes/DABEST/meta.yaml (hunk 4 above)
Comment (synapticarbors) [reply recipes/DABEST/meta.yaml]: It looks like the `0.1.1` tarball now has the license, so can you delete the license from the recipe directory and change this line to `license_file: LICENSE`. 

After that, everything looks good to merge.

Comment (scopatz) [reply recipes/DABEST/meta.yaml]: pinging this!

Comment (josesho) [reply recipes/DABEST/meta.yaml]: sorry, was out of the office, done!

File: recipes/DABEST/meta.yaml (hunk 2 above)
Comment (synapticarbors) [reply recipes/DABEST/meta.yaml]: Sorry for the delay responding to this. If, for example, `seaborn` needed to strictly be at version `0.8`, you would change:

```
   - seaborn
```

to 

```
   - seaborn ==0.8
```

See examples in the following:

https://conda.io/docs/user-guide/tasks/build-packages/package-spec.html#examples

Comment (josesho) [reply recipes/DABEST/meta.yaml]: OK, done!

File: recipes/DABEST/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - matplotlib==2.1
```
Comment (synapticarbors) [inline recipes/DABEST/meta.yaml]: You need a space between the package name and the version spec -- e.g. `matplotlib ==2.1`

Comment (josesho) [reply recipes/DABEST/meta.yaml]: ok, thanks! fixed!

File: recipes/DABEST/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - matplotlib ==2.1
+    - seaborn ==0.8
+    - pandas ==0.22
+    - numpy ==1.13
+    - scipy ==1.0
```
Comment (scopatz) [inline recipes/DABEST/meta.yaml]: I don't think you need to pin things like numpy, matplotlib, etc to certain versions here.... If there are version restrictions, it is better to use `numpy >= 1.11`

**Pull request closed without merging**